deployments/liqo/subcharts/discoveryOperator/templates/discovery-deploy.yaml

```diff
@@ -9,6 +9,148 @@ metadata:
     app: liqo.io
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: liqo-discovery-role
```

Comment:
My opinion is that `-role` in the name is a bit wierd (no one seems doing that). Just an opinion though.